### PR TITLE
testing svm, naive bayes, random forest models

### PR DESCRIPTION
**Pull request by ajayboby**
These files include testing  accuracy of different models on given data set

### CODE REVIEW

File: Background Reserach/Code Research/test_naive_bayes.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		rm_punct = remove_punct(rm_st)
+		train.at[i,'sentence'] = ' '.join(rm_punct)
+		# print(train.sentence[i])
+	for i in range(0,1):
+		# print(test.sentence[i])
+		sentence_tokens = get_list_tokens(test.sentence[i])
+		rm_st = remove_stop_words(sentence_tokens)
+		rm_punct = remove_punct(rm_st)
+		test.at[i,'sentence'] = ' '.join(rm_punct)
+		# print(test.sentence[i])
```
Comment (c1531958) [inline Background Reserach/Code Research/test_naive_bayes.py:74]: replace this with 

train['sentence'] = preprocess(train)
test['sentence'] = preprocess(test)

File: Background Reserach/Code Research/test_naive_bayes.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	for token in list_tokens:
+		if token not in english_stop_words:
+			clean_list_tokens.append(token)
+
+	return clean_list_tokens
+
+def remove_punct(list_tokens):
+	no_punct = [i for i in list_tokens if i not in punct]
+	return no_punct
+
```
Comment (c1531958) [inline Background Reserach/Code Research/test_naive_bayes.py:40]: This has been moved into a preprocess file

File: Background Reserach/Code Research/test_naive_bayes.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import nltk
+from nltk.corpus import stopwords
+from sklearn.metrics import accuracy_score, precision_recall_fscore_support
```
Comment (c1531958) [inline Background Reserach/Code Research/test_naive_bayes.py:3]: Can be replaced with import error_stats
The can be used:

stats = error_stats.get_stats(test.label, prediction)
stats_all.append(stats)

File: Background Reserach/Code Research/test_naive_bayes.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+	ax = error_stats.format_conf_matrix(train, test, prediction, word, words)
+	# get stats (accuracy, precision etc)
+	stats = error_stats.get_stats(test.label, prediction)
+	stats_all.append(stats)
+
+t1 = time.time()
+print('Time it took: {}'.format(t1 - t0))
+
+df = pd.DataFrame(stats_all, columns=['accuracy', 'precision', 'recall', 'fscore'], index=words)
+print(df)
```
Comment (c1531958) [inline Background Reserach/Code Research/test_naive_bayes.py:106]: Also below can add the line plt.show() to plot the confusion matrixes if error_stats file has been imported
